backend/app/core/social_service.py

```
@@ -1,297 +1,346 @@
-        
+
         logger.info(f"Deleted post {post_id}")
         return True
-    
-    def get_social_feed(self, user_id: int, feed_params: SocialFeedParams) -> Tuple[List[SocialPost], int]:
+
+    def get_social_feed(
+        self, user_id: int, feed_params: SocialFeedParams
+    ) -> Tuple[List[SocialPost], int]:
         """Get personalized social feed"""
         query = self.db.query(SocialPost).filter(SocialPost.is_hidden == False)
```

Ruff (0.11.9)

Avoid equality comparisons to `False`; use `if not SocialPost.is_hidden:` for false checks

Replace with `not SocialPost.is_hidden`

(E712)